### ISSUE TITLE
[Feature Request] Add to CanRawFilter an additional "inverted" output

### ISSUE THREAD

**Issue by jogo-** [GENIVI/CANdevStudio#155]
It should be helpful to add to the CanRawFilter component an additional "inverted" output for frames unmatched by any filter.
It could be marked as : ¬x (NOT x, Nx or !x)

**Comment by rkollataj** [comment]
Do you mean to for eample set "NOT 0x123" and match all frames different than 0x123?

**Comment by jogo-** [comment]
No, I mean in the CanRawFilter node, add an additional "inverted" output (below the standard "RAW" one) for frames unmatched by any filter.
So, this second output could be used to link/cable to other nodes for frames unmatched by any filter.

**Comment by rkollataj** [comment]
Oh I see. The thing is that CanRawFilters produces more like accepted/dropped rather than matched/unmatched frames. If there is no appropriate filter in a table then frame is being matched by default fiter (defined below the table). So each frame is in fact matched and is accepted or dropped. Adding another output that will pass over dropped frames seems confusing. 

What is exact use case you are interested in? There may be some other way to achieve it.

**Comment by jogo-** [comment]
Using just one CanRawFilter, you could split between the 2 outputs the diagnostic CAN frames on one hand and the other CAN frames on the other hand,, for example.
